data/core/units/orcs/Shaman.cfg

```diff
@@ -0,0 +1,99 @@
+#textdomain wesnoth-units
+
+[unit_type]
+    id=Orcish Shaman
+    name=_"Orcish Shaman"
+    race=orc
+    image="units/orcs/shaman.png"
+    profile="portraits/orcs/shaman.webp"
+    hitpoints=23
```

Comment:
HP value is really low for a level 2 mage type

Reply:
raised to 32 in 28ef66c